data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/command/AddPreAuthToken.avsc

```diff
@@ -0,0 +1,34 @@
+{
+  "type": "record",
+  "name": "AddPreAuthToken",
+  "namespace": "net.corda.data.membership.db.request.command",
+  "doc": "Add a pre auth token.",
+  "fields": [
+    {
+      "name": "tokenId",
+      "doc": "Token identifier of the pre auth token.",
+      "type": "string"
+    },
+    {
+      "name": "ownerX500Name",
+      "doc": "X500 name of the owner of the pre auth token.",
+      "type": "string"
+    },
+    {
+      "name": "ttl",
+      "doc": "A timestamp for when the pre auth token expires.",
+      "type": [
+         "null",
```

Comment:
Just want to double check if this is intentionally nullable? Checking because I think the design doc said it would default if it wasn't set.

Reply:
Yes. There is some discussion here: https://github.com/corda/corda-runtime-os/pull/2946#discussion_r1082457191.

Reply:
Looks like I forgot to update the entity 🤦 (I will leave a comment on the other PR).